### ISSUE TITLE
Ensure grant funding is nicely represented in schemas

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#4293]
xref https://schema.org/Grant

```
  "funding": {
     "@type": "Grant",
     "identifier": "1448821",
     "funder": {
       "@type": "Organization",
       "name": "National Science Foundation",
       "identifier": "https://doi.org/10.13039/100000001"
     }
   }
```

**Comment by hechth** [comment]
@hexylena I didn't really figure out how to do the reference to the funder properly in the XSD representation - any idea on how to do that properly?

**Comment by hexylena** [comment]
@hechth the XSD representation? Could you point me to where that would be? We're not using XML for much. This isssue is specifically about the LD+JSON schemas we use on e.g. tutorial pages.

**Comment by hexylena** [comment]
fixed in #4408
